monai/transforms/croppad/array.py

```diff
@@ -160,62 +207,42 @@ class SpatialPad(Transform):
 
     """
 
-    backend = Pad.backend
-
     def __init__(
         self,
         spatial_size: Union[Sequence[int], int],
         method: Union[Method, str] = Method.SYMMETRIC,
```

Comment:
please help simplify the type hint in other places as well.
```suggestion
        method: str = Method.SYMMETRIC,
```